applications/ray/kuberay-tpu-webhook/main.go

```diff
@@ -413,10 +413,13 @@ func mutatePod(admissionReview *admissionv1.AdmissionReview) (*admissionv1.Admis
 		numOfHosts, _ := getNumTPUHostsFromTopology(topology, acceleratorType) // ignore error here because topology may not be set yet
 		replicaIndex := getReplicaIndex(clusterName)
 		podSlice := slice{clusterName, groupName, replicaIndex, numOfHosts}
-		tpuWorkerID := getNextWorkerID(podSlice, replicaIndex)
+		tpuWorkerID := 0 // defaults to 0 for single-host
```

Comment:
Any way to add a unit test?

Reply:
I'm working on adding a unit test suite for the webhook that'll cover this, it should be out by the end of the week.